app/assets/stylesheets/lostpassword.scss

```diff
@@ -0,0 +1,128 @@
+*{
```

Comment:
don't use a lot of CSS, but directly (bootstrap classes](https://getbootstrap.com/docs/4.3/getting-started/introduction/)